config/routes.rb

```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
   resources :boards do
     resources :lists, only: [:new, :create, :edit, :show, :destroy, :update]
+      resources :tickets, only: [:new, :create]
```

Comment:
buuuuuu
`Resources should never be nested more than 1 level deep.` mowilem o tym 100 razy :)